public/layout.css

```diff
@@ -6,3 +6,23 @@ body {
 .navbar {
   margin-bottom: 20px;
 }
+
```

Comment:
Just a small nitpick here.  Spacing seems to be a little off.  It looks like tabs were introduced and the entry above only uses two space for indent.  I'm gonna go ahead and approve the pull request though